jekyll/_cci2/env-vars.md

```diff
@@ -82,18 +82,29 @@ jobs:
 
 ### Interpolating Environment Variables 
 
-CircleCI does not support interpolation of environment variables.
+CircleCI does not support interpolation
+when defining configuration variables like `working_directory` or `images`.
 All defined values are treated literally.
 
-One workaround is to export the required variable within a command.
+However, it is possible to interpolate a variable within a command
+by setting it for the current shell.
 
 ```yaml
-- run:
-    command: |
-      export PATH=/go/bin:$PATH
-      go-get ...
+version: 2
+jobs:
+  build:
+    steps:
+      - run: echo 'export PATH=/path/to/foo/bin:$PATH' >> $BASH_ENV  # updating PATH
+      - run: echo 'export VERY_IMPORTANT=$(cat important_value)' >> $BASH_ENV  # runtime definition of env var
```

Comment:
looking at the react-native config.yml they also add 

 -run: source $BASH_ENV

Reply:
Hmm, that seems odd to me. I would understand if it were sourced in the same `run` step as the new environment variable. But my understanding is that `$BASH_ENV` is automatically loaded anyway in each `run` step.

Is [this](https://github.com/CircleCI-Public/circleci-demo-react-native/blob/master/.circleci/config.yml) the react-native `config.yml` you meant? I can't find the line you mentioned. :(